### ISSUE TITLE
Rate the alignment of a contribution

### ISSUE THREAD

**Issue by aahna-ashina** [sector-3/protocol#51]
Allow rating (attestation) of existing contributions.

Depends on #20 being implemented first.

**Comment by developerfred** [comment]
@aahna-ashina Does the rating have any impact on contributions or is it just additional information for contributor metrics?

**Comment by aahna-ashina** [comment]
@developerfred Yes, the average `alignmentPercentage` will be used for the final calculation.  So other contributors can indicate that they think a contribution has lower, equal or higher alignment.

Example:  If a contributor rates his/her contribution as "mostly aligned," while two other contributors rate it as "highly aligned," the final reward allocation weighting will be "highly aligned."
